test/src/test/java/jenkins/security/Security218CliTest.java

```diff
@@ -254,6 +254,7 @@ public void checkRoles(RoleChecker checker) throws SecurityException {
                     }
                 });
             } catch (Exception ex) {
+                ex.printStackTrace();
```

Comment:
looks like this will dump out stack traces even for a successful test (which is bad) as stack traces for OK tests look scary when running and set of triggers in human brains.

Reply:
Generally agree. In this case it is a security test and if you are running it interactively (i.e., `-Dtest=…`) you should be aware that an exception is wanted here. I specifically added this line because I had trouble diagnosing a problem in the test previously, and this made things much clearer.